Project/zepeto-modules/Assets/Zepeto ScreenShot Module/ZepetoScript/SelfieCamera.ts

```diff
@@ -125,12 +129,38 @@ export default class SelfieCamera extends ZepetoScriptBehaviour {
     }
 
     LateUpdate() {
-
-
         if (this.currentTarget == null || this.targetLookAt == null)
             return;
 
         //this.CameraInput();
         this.CameraMovement();
     }
+    
+    // A function to adjust the selfie camera;
+    AdjustSelfieCamera()
+    {
+        this._screenShotModeManager = Object.FindObjectOfType<ScreenShotModeManager>();
+        const initialHeight = this.height;
+        this.height = this.getSelfieCameraHeight(this._screenShotModeManager.localPlayer)
+        
+        // Check if the character is too tall and adjust the camera distance accordingly
+        if(this.height > 0.9)
+        {
+            this.distance = this.distance / this.height;
+        }
+        else
+        {
+            this.distance = this.distance / (this.height * initialHeight);
+        }
+    }
+
+    // This function returns the camera height following the head position;
+    private getSelfieCameraHeight(localPlayer:ZepetoPlayer): number {
```

Comment:
What do you think of clarifying that this returns a 'new', 'processed' value for the selfie camera height?

I suggest `calculateSelfieCameraHeight`.

Reply:
Oh yeah, this makes more sense. I'll update this as soon as I can.